database/migrations/2021_02_01_110000_add_auth_permissions.php

```diff
@@ -9,7 +9,7 @@ class AddAuthPermissions extends BasePermissionsMigration
     public function up()
     {
         $permissions = [
-            'access admin' => ['Owner','Executive','Staff','Customer'],
```

Comment:
Giving `Customer` the `access admin` permission doesn't make any sense to me, so I removed it.

Reply:
At some point, I want them to be able to login and see their order history, but we can build that outside of Nova.

Reply:
The policies in checkout allow for this access via the API routes in the package.  

Order Policy allows view any to all users - 
https://github.com/tipoff/checkout/blob/cecbeef5f8d5d782fa23abf6d4b1a5db4ea94f2e/src/Policies/OrderPolicy.php#L17

Coupled with a scoping in the controller -
https://github.com/tipoff/checkout/blob/cecbeef5f8d5d782fa23abf6d4b1a5db4ea94f2e/src/Http/Controllers/Api/OrderController.php#L27

Similar for show, but the policy is checking for order ownership.

This allows building complete order history support outside of Nova (I'd personally never want customers able to logon to my admin tools, just seems like too much possibility for a mistake, something left unguarded that should be)